slime/rollout/sglang_example.py

```diff
@@ -38,6 +38,13 @@ def __init__(self, args):
             no_stop_trim=True,
             spaces_between_special_tokens=False,
         )
+
+        # Add token output mode configuration
+        if getattr(args, "use_token_output", False):
+            self.use_token_output = True
+        else:
+            self.use_token_output = False
+        print(self.sampling_params)
```

Comment:
please remove the print here or format the print with messages.

also, since we have store the `self.args`, it seems that using `self.args.use_token_output` directly would be cleaner.